configs/testing/metrics/dbscan.json

```diff
@@ -0,0 +1,174 @@
+{
+    "common": {
+        "lib": ["sklearn"],
+        "data-format": ["pandas"],
+        "data-order": ["F"],
+        "dtype": ["float64"]
+    },
+    "cases": [
+        {
+            "algorithm": "dbscan",
+            "dtype": ["float64"],
+            "dataset": [
+                {
+                    "source": "npy",
+                    "name": "ijcnn",
+                    "training":
+                    {
+                        "x": "data/ijcnn_x_train.npy",
+                        "y": "data/ijcnn_y_train.npy"
+                    },
+                    "testing":
+                    {
+                        "x": "data/ijcnn_x_test.npy",
+                        "y": "data/ijcnn_y_test.npy"
+                    }
+                }
+            ],

```

Comment:
And what can be done - but I'm not sure about the need, that we can enumerate datasets in a list in, say, single test case. This will make the code much shorter